spec/order_spec.rb

```diff
@@ -0,0 +1,49 @@
+require 'menu'
+require 'order'
+
+describe Order do
+
+  let(:menu) { double :menu }
+  let(:dish) { double :dish }
+  # let(:menu) { object_double(Menu.new, :has_dish? => true) }
+
+  subject { described_class.new(menu) }
```

Comment:
You get this (`subject`) 'for free' when you refer to the class under test as a constant `describe Order do`.  You only need to create the subject explicitly if you want to initialize it with args.